### PR TITLE
Add introduction to usage of CSS in GettingStarted.md

### PR DESCRIPTION
**Pull request by simplePCBuilding**
As discussed in #860, here are some changes to make the guide more complete, with also including some notes on CSS.

### CODE REVIEW

**Comment by simplePCBuilding** [comment]
I will be working on some more on impress.js starting in November / December this year as I will again have time to work on other projects. Expect some cool PRs (I've got a few ideas)

File: README.md [hunk 1]
```diff
@@ -22,9 +22,6 @@ You can include this link directly inside of your HTML file in its header. If yo
 - V1.1.0: https://cdn.jsdelivr.net/gh/impress/impress.js@1.1.0/js/impress.js
 - Source: https://cdn.jsdelivr.net/gh/impress/impress.js/js/impress.js
 
-### Getting Started Guide
-Check out our new [Getting Started](GettingStarted.md) guide if you want a quick introduction to the project!
-
```
Comment (henrikingo) [inline README.md:27]: Why is this removed??

**Review by henrikingo — COMMENTED** [review]
Sorry but I don't get this PR? Why add more text for such a specific case? 

The guide instructs to use impress-common.css. If yolu leave it out, of course there will be consequences.

Similarly for the fix, why is the fix to add back CSS that you already deleted? Why isn't the fix to remove the fallback message entirely? (For example.)

I think I will close this. Sorry. But I don't see that this is needed.
